Resources/Prototypes/_RMC14/Entities/Mobs/training_dummy.yml

```
@@ -0,0 +1,42 @@
+- type: entity
+  parent: CMBaseMobSpeciesOrganic
+  id: RMCTrainingDummy
+  name: training dummy
+  description: A training dummy used by the UNMC to simulate a humanoid body. Feels almost too lifelike.
+  suffix: RMC14
+  components:
+  - type: Loadout
+    prototypes: [ RMCGearTrainingDummy ]
+  - type: FixedIdentity
+    name: cm-chatsan-replacement-synthetic
+  - type: RMCTrainingDummy
+    removeComponents:
+    - type: CanEnterCryostorage
+    - type: Hunger
+    - type: Infectable
+    - type: NewPlayerLabel
+    - type: Perishable
+    - type: SSDIndicator
+    - type: TacticalMapTracked
+    - type: Thirst
+    - type: Vocal
```

```suggestion
    - type: Vocal
    - type: MindContainer
```
will remove the SSD text and icon